oraclizeAPI_0.5.sol

```diff
@@ -1013,16 +6214,30 @@ contract usingOraclize {
         return mint;
     }
 
-    function parseInt(string memory _a) internal pure returns (uint _parsedInt) {
+    function parseInt(string memory _a)
+        internal
+        pure
+        returns (uint256 _parsedInt)
+    {
         return parseInt(_a, 0);
     }
 
-    function parseInt(string memory _a, uint _b) internal pure returns (uint _parsedInt) {
+    function parseInt(
+        string memory _a,
+        uint256 _b
+    )
+        internal
+        pure
+        returns (uint256 _parsedInt)
+    {
         bytes memory bresult = bytes(_a);
-        uint mint = 0;
+        uint256 mint = 0;
         bool decimals = false;
-        for (uint i = 0; i < bresult.length; i++) {
-            if ((uint(uint8(bresult[i])) >= 48) && (uint(uint8(bresult[i])) <= 57)) {
+        for (uint256 i = 0; i < bresult.length; i++) {
+            if (
+                (uint(uint8(bresult[i])) >= 48) &&

```

Comment:
uint256 especially here, as @riccardopersiani pointed out would help this make more sense.

Reply:
It tells me that you're looking at an outdated commit. All the `uint`s are now explicit.